tubes/lua.lua

```diff
@@ -229,11 +229,13 @@ end
 -------------------------
 
 local function safe_print(param)
-	local string_meta = getmetatable("")
-	local sandbox = string_meta.__index
-	string_meta.__index = string -- Leave string sandbox temporarily
-	print(dump(param))
-	string_meta.__index = sandbox -- Restore string sandbox
+	if mesecon.setting("luacontroller_print_behavior", "log") == "log" then
```

Comment:
Let's call this luatube instead :)
```suggestion
	if mesecon.setting("luatube_print_behavior", "log") == "log" then
```

Or this (after actual node base name lua_tube, adjust settingtypes.txt below accordingly):
```suggestion
	if mesecon.setting("lua_tube_print_behavior", "log") == "log" then
```

Would be also good to add this to settingtypes.txt for GUI config.
Mesecons has this (just changed name), could probably just go with it:
```
# Behavior of print() inside a lua tube. By default, this emits a message into actionstream.
# Set it to noop if you wish to disable that behavior.
mesecon.luatube_print_behavior (Behavior of print) enum log log,noop
```

Reply:
Actual node base name is lua_tube but I'd say calling it luatube in settings is just fine too, however you (or someone else) likes to have it.

Reply:
Instead of abusing Mesecon's namespace, I added a new setting, `pipeworks_lua_tube_print_behavior`, controlling Lua tube's behavior. To handle such text input, there are changes in `default_settings.lua`.

Done in 1980ae1b719b6677c79cba60c8a39e3094100de3